addons/account/models/res_config_settings.py

```diff
@@ -191,6 +191,22 @@ class ResConfigSettings(models.TransientModel):
         domain="[('deprecated', '=', False), ('account_type', 'in', ('income', 'income_other', 'expense'))]",
     )
 
+    # Accounts for allocation of discounts
+    account_discount_income_allocation_id = fields.Many2one(
+        comodel_name='account.account',
+        string='Vendor Bills Discounts Account',
+        readonly=False,
+        related='company_id.account_discount_income_allocation_id',
+        domain="[('account_type', 'in', ('income', 'expense'))]",
```

Comment:
Why expense?

Reply:
I can't speak for every Accounting practice in the world, but I've seen people using negative line in Sales to reduce amounts (for example in depreciations or cash discounts). I think we should not restrict the choice too much here.

Reply:
So you would have a line on an income account, and then you would do a "negative" discount on an expense account? :exploding_head:
How would that work regarding the total sum D-C anyways?

Reply:
But I think I understand what you mean, ok for keeping it then